website_sale_cart_selectable/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+    
+============================
+Website Sale Cart Selectable
+============================
+
+This module was written to extend the functionality of the website to control
```

Comment:
This module extends the functionality of the website to improve the control on the
button "Add to cart". Button  "Add to cart" can now be set to be published or not.